app/models/feed.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+class Feed < ActiveRecord::Base
+  include HasExternalUrls
+
+  has_and_belongs_to_many :pages
+
+  validates :name, presence: true, uniqueness: true
+  validates :url, presence: true, uniqueness: true
+
+  acts_as_url :name, url_attribute: :slug, only_when_blank: true, allow_duplicates: false
+
+  scope :tumblr, -> { where('url LIKE (?)', '%tumblr.com%') }
```

Comment:
I'm not too keen on this duplication of the URL patterns already present in the `HasExternalUrls` concern.

Where you have used `landing_page.feeds.tumblr.first` and `landing_page.feeds.blog.first`, could we just use `landing_page.feeds.detect(&:tumblr?)` and `landing_page.feeds.detect(&:europeana_blog?)` instead, and remove these scopes?